ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: "off" */
```

Comment:
Ah yeah my bad I forgot about that, totally makes sense to leave for now until we finish this off.